src/buffer_manager.ts

```diff
@@ -722,24 +728,27 @@ export class BufferManager implements Disposable {
             // !Another hack is to retrieve cursor with delay - when we receive an external buffer the cursor pos is not immediately available
             // [1, 0]
             setTimeout(async () => {
-                const neovimCursor: [number, number] = await this.client.request("nvim_win_get_cursor", [closeWinId]);
-                if (neovimCursor) {
```

Comment:
I hesitated here as well, for the same reason as the other one. However, the nvim doc doesn't say this can return anything nullish, so I'm inclined to think this was an erroneous error check as well?